lib/kafo/kafo_configure.rb

```diff
@@ -157,46 +207,14 @@ def execute
       return self
     end
 
-    def self.run
-      return super
-    rescue SystemExit
-      self.exit_handler.exit(self.exit_code) # fail in initialize
-    end
-
-    def self.exit(code, &block)
-      exit_handler.exit(code, &block)
-    end
-
-    def self.exit_code
-      self.exit_handler.exit_code
-    end
-
-    def self.in_help_mode?
-      ARGV.include?('--help') || ARGV.include?('--full-help') || ARGV.include?('-h')
-    end
-
     def exit_code
       self.class.exit_code
     end
 
-
     def help
       self.class.help(invocation_path, self)
     end
 
-    def self.help(*args)
-      kafo          = args.pop
-      builder_class = kafo.full_help? ? HelpBuilders::Advanced : HelpBuilders::Basic
-      args.push builder_class.new(kafo.params)
-      super(*args)
-    end
-
-    def self.app_option(*args, &block)
-      self.app_options ||= []
-      self.app_options.push self.option(*args, &block)
-      self.app_options.last
-    end
```

Comment:
This option (or rather, the moved one) is still used in HookContext, which refers to `self.kafo.class.app_option`

Reply:
Yep, that's intended not to change, as that corresponds to the defined method here: https://github.com/theforeman/kafo/pull/299/files#diff-0598f0904f7f79914887ee5bf9fee84f5f21e599bdba5e885a6d46b9da703505R9 which is included in KafoConfigure making the option available.